Compilation error when having serde_json as a dependency

```rust
error[E0283]: type annotations needed
  --> /home/jonas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_replicon-0.11.0/src/client.rs:57:29
   |
57 |                         let end_pos = message.len().try_into().unwrap();
   |                             ^^^^^^^
...
85 |                         if cursor.position() == end_pos {
   |                                              -- type must be known at this point
   |
   = note: multiple `impl`s satisfying `u64: PartialEq<_>` found in the following crates: `core`, `serde_json`:
           - impl PartialEq for u64;
           - impl PartialEq<serde_json::value::Value> for u64;
help: consider giving `end_pos` an explicit type
   |
57 |                         let end_pos: /* Type */ = message.len().try_into().unwrap();
   |                                    ++++++++++++
```

Note that serde_json is included with default bevy features (bevy_gltf)

--- a/src/client.rs
+++ b/src/client.rs
@@ -54,7 +54,7 @@ impl ClientPlugin {
             world.resource_scope(|world, mut entity_map: Mut<NetworkEntityMap>| {
                 world.resource_scope(|world, replication_rules: Mut<ReplicationRules>| {
                     while let Some(message) = client.receive_message(REPLICATION_CHANNEL_ID) {
-                        let end_pos = message.len().try_into().unwrap();
+                        let end_pos: u64 = message.len().try_into().unwrap();
                         let mut cursor = Cursor::new(message);
 
                         if !deserialize_tick(&mut cursor, world)? {

